optimizer_wrapper.rb

```diff
@@ -648,6 +662,7 @@ def self.compute_route_total_dimensions(vrp, route, matrix)
           activity[:current_distance] ||= total[dimension].round if dimension == :distance
         }
       end
+      next if point.nil?
```

Comment:
Do we need this? It looks like to me that to skip one `previous = nil` assignment, we will do hundereds of extra `point.nil?` checks which is not necessary in the first place because `previous && point` handles the `nil` case correctly

Reply:
We do need this line, otherwise, the previous_point is updated with a nil value even if it is a break

Reply:
You say bug, I say feature :p